bin.src/run_generic.py

```diff
@@ -85,6 +75,8 @@ def parseArgs(subdir='out', parser=None):
                                                                  "Default is runName/%s." % (subdir))
     parser.add_argument('--plotOnly', dest='plotOnly', action='store_true',
                         default=False, help="Reload the metric values from disk and re-plot them.")
+    parser.add_argument('--sqlConstraint', type=str, default=None,
```

Comment:
Would it be useful to add an example of the type of additional sql constraint that can be passed to the batches to show the right format and possibilities?